publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+# Results of accessibility self-review for Web of Things (WoT) Architecture
+
+Answers to accessibility (FAST) review checklist [(from here)](https://w3c.github.io/apa/fast/checklist.html)
+for the Web of Things (WoT) architecture as a whole.
```

Comment:
Please add a link to the architecture spec.

Reply:
Here I was using the word "architecture" to refer to all deliverables but this is confusing.  I will change this to "deliverables as a whole" but ALSO add links to *all* deliverables.